src/io/flutter/module/FlutterModuleBuilder.java

```diff
@@ -137,6 +137,14 @@ public Module commitModule(@NotNull Project project, @Nullable ModifiableModuleM
     return flutter;
   }
 
+  private static String validateSettings(FlutterCreateAdditionalSettings settings) {
+    final String description = settings.getDescription();
+    if (description != null && description.contains(": ")) {
```

Comment:
Would just a `':'` be an error? If so, perhaps shorten from checking for `': '`.

Reply:
Nope.  Steve pointed out that it needs the trailing space to be (mis)treated as a delimiter.